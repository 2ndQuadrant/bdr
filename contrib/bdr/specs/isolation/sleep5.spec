conninfo "node1" "dbname=node1"
conninfo "node2" "dbname=node2"
conninfo "node3" "dbname=node3"

session "snode1"
step "s1sleep" { SELECT pg_sleep(5); }

session "snode2"
step "s2sleep" { SELECT pg_sleep(5); }

session "snode3"
step "s3sleep" { SELECT pg_sleep(5); }

permutation "s1sleep" "s2sleep" "s3sleep"
