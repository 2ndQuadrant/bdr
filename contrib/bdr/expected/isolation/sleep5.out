Parsed test spec with 3 sessions

starting permutation: s1sleep s2sleep s3sleep
step s1sleep: SELECT pg_sleep(5);
pg_sleep       

               
step s2sleep: SELECT pg_sleep(5);
pg_sleep       

               
step s3sleep: SELECT pg_sleep(5);
pg_sleep       

               
